lms/djangoapps/courseware/views/index.py

```diff
@@ -79,7 +79,11 @@
 from ..model_data import FieldDataCache
 from ..module_render import get_module_for_descriptor, toc_for_course
 from ..permissions import MASQUERADE_AS_STUDENT
-from ..toggles import COURSEWARE_MICROFRONTEND_COURSE_TEAM_PREVIEW, should_redirect_to_courseware_microfrontend
+from ..toggles import (
+    COURSEWARE_MICROFRONTEND_COURSE_TEAM_PREVIEW,
+    REDIRECT_TO_COURSEWARE_MICROFRONTEND,
+    should_redirect_to_courseware_microfrontend
```

Comment:
Nitpick, but I advocate for placing trailing commas anywhere syntactically equivalent [1];
it makes maintaining a fork easier ;)

- [1] https://github.com/Stanford-Online/edx-platform/wiki/collections-trailing-comma

Reply:
Done. Should our linter doesn't enforce this?